pkg/controllers/machinesetsync/machineset_sync_controller.go

```diff
@@ -121,7 +125,7 @@ func (r *MachineSetSyncReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 // Reconcile reconciles CAPI and MAPI MachineSets for their respective namespaces.
 func (r *MachineSetSyncReconciler) Reconcile(ctx context.Context, req reconcile.Request) (ctrl.Result, error) {
-	logger := log.FromContext(ctx).WithValues("namespace", req.Namespace, "name", req.Name)
+	logger := log.FromContext(ctx)
```

Comment:
Why not include the name and namespace on every log?

Reply:
So I noticed when looking at test output we're already getting them from somewhere - i'm not sure where it's set up but this ends up being extra noise :/ 

```
I1107 15:12:35.353217   96821 machineset_sync_controller.go:161] "MAPI machine set not found"
controller="machineset" controllerGroup="machine.openshift.io" controllerKind="MachineSet"
MachineSet="openshift-machine-api-v4752/foo" namespace="openshift-machine-api-v4752" name="foo"
reconcileID="152f0deb-bb27-4ba5-a542-2e3e9ca914cc"
```

Reply:
I do wonder if this is coming from how we bootstrap envtest,  but haven't seen anything obvious.

Reply:
Looks like controller runtime internals 🤔 I'd potentially still include the name and namespace tbh